adapters/appnexus/appnexus.go

```diff
@@ -354,14 +356,56 @@ func (a *AppNexusAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *ada
 	}
 	reqExt.Appnexus.IsAMP = isAMP
 	reqExt.Appnexus.HeaderBiddingSource = a.hbSource + isVIDEO
+
+	imps := request.Imp
+
+	// For long form requests adpod_id must be sent downstream.
+	// Adpod id is a unique identifier for pod
+	// All impressions in the same pod must have the same pod id in request extension
+	// For this all impressions in  request should belong to the same pod
+	// If impressions number per pod is more than maxImpsPerReq - divide those imps to several requests but keep pod id the same
+	if isVIDEO == 1 {
+		podImps := groupByPods(imps)
+
+		requests := make([]*adapters.RequestData, 0, 0)
```

Comment:
Please set the initial capacity to `len(podImps)` to minimize realloactions.

Reply:
Done